src/clib/lib/enkf/enkf_state.cpp

```diff
@@ -32,15 +32,13 @@ void enkf_state_initialize(rng_type *rng, enkf_fs_type *fs,
         enkf_node_store(param_node, fs, node_id);
 }
 
-ecl_sum_type *load_ecl_sum(const ecl_config_type *ecl_config,
-                           const char *run_path, const char *eclbase) {
+ecl_sum_type *load_ecl_sum(const char *run_path, const char *eclbase) {
     ecl_sum_type *summary = NULL;
 
-    const bool fmt_file = ecl_config_get_formatted(ecl_config);
     char *header_file = ecl_util_alloc_exfilename(
-        run_path, eclbase, ECL_SUMMARY_HEADER_FILE, fmt_file, -1);
+        run_path, eclbase, ECL_SUMMARY_HEADER_FILE, DEFAULT_FORMATTED, -1);
```

Comment:
So I would expect here that `ecl_config_get_formatted` and `ecl_config_type->io_config` would be deleted in this commit.

Reply:
Ah, now I understand, the entire file is deleted later, nvm.